fields.go

```diff
@@ -134,3 +142,13 @@ func (f *FieldWriter) Uint64(code byte, v uint64) {
 func (f *FieldWriter) Timestamp(code byte, t time.Time) {
 	f.Uint64(code, uint64(t.UnixNano()/1000))
 }
+
+func (f *FieldWriter) String(code byte, v string) {
+	// TODO: Check length of string?
```

Comment:
It might be good to have this return an error if the string is longer than 255 chars